src/screens/add-transaction-screen/quick-add-accounts-selector.tsx

```diff
@@ -80,13 +81,40 @@ export const QuickAddAccountsSelector = connect(
     loading,
     refetch,
   } = useLedgerMeta(userId);
+
   const [selectedAssets, setSelectedAssets] = useState(
     assetsOptionTabs.length > 0 ? assetsOptionTabs[0].options[0] : ""
   );
   const [selectedExpenses, setSelectedExpenses] = useState(
     expensesOptionTabs.length > 0 ? expensesOptionTabs[0].options[0] : ""
   );
 
+  React.useEffect(() => {
+    async function init() {
+      try {
+        const lastSelectedAssets = await AsyncStorage.getItem(
```

Comment:
can we turn these getter and setter to react hooks? https://usehooks.com/useLocalStorage/